lib/workers/repository/errors-warnings.ts

```diff
@@ -111,7 +112,9 @@ export function getDepWarningsDashboard(
   }
 
   const depWarnings = warnings
-    .map((w) => w.replace('Failed to look up dependency ', ''))
+    .map((w) =>
+      w.replace(regEx(/^Failed to look up(?: [-\w]+)? dependency /), '')
+    )
```

Comment:
Would be made a separate PR, but meh

Reply:
yes, please do this in a seperate PR, it's unrelated